packages/backend/server/src/models/notification.ts

```diff
@@ -209,7 +209,7 @@ export class NotificationModel extends BaseModel {
       where: {
         userId,
         ...(options?.includeRead ? {} : { read: false }),
-        ...(options?.after ? { createdAt: { gt: options.after } } : {}),
+        ...(options?.after ? { createdAt: { lt: options.after } } : {}),
```

Comment:
This is strange. Are you sure?

Reply:
fixed in https://app.graphite.dev/github/pr/toeverything/AFFiNE/11430/fix(server)-correct-notification-pagination-by-using-'lt'-instead-of-'gt'-for-createdAt